interface/main/tabs/menu/menus/patient_menus/standard.json

```diff
@@ -0,0 +1,98 @@
+[
+    {
+        "label": "History",
+        "menu_id": "history",
+        "target": "main",
+        "on_click":"top.restoreSession()",
+        "url": "../history/history.php",
+        "pid":"false",
+        "global_controlled":"",
```

Comment:
Note the global control should be the same as the main menu (ie. no need to reinvent the wheel on this mechanism), which is:
global_req

Reply:
Also, will not need target or requirement in these entries.

Reply:
also, place the global_req as the last item in this (after acl_req) to be able to support nesting menu down the road (ie. it needs to be after the children entry).